libkbfs/disk_block_cache.go

```diff
@@ -139,7 +141,9 @@ func newDiskBlockCacheStandardFromStorage(
 			closer()
 		}
 	}()
-	blockDb, err := openLevelDB(blockStorage)
+	blockDbOptions := *leveldbOptions
+	blockDbOptions.CompactionTableSize = defaultBlockCacheTableSize
```

Comment:
This variable is eventually plumbed through here, so I _think_ it does what we need.
https://github.com/syndtr/goleveldb/blob/3c5717caf1475fd25964109a0fc640bd150fce43/leveldb/db_compaction.go#L399-L401

I haven't tested it yet though.